src/__internal__/validation-message/validation-message.component.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+import StyledValidationMessage from "./validation-message.style";
+
+const ValidationMessage = ({ error, warning }) => {
+  const validation = error || warning;
+  const isStringValidation = typeof validation === "string";
+
+  return (
+    isStringValidation && (
```

Comment:
This component will return `false` when it isn't string validation, we should probably move this out of the return and explicitly return null if it's false 

```js

if (isStringValidation) {
  return (...)
}

return null;
```

Reply:
yeah good point 👍